mne_bids/path.py

```diff
@@ -762,31 +840,47 @@ def _find_best_candidates(params, candidate_list):
     return best_candidates
 
 
-def _get_bids_fname_from_filesystem(*, bids_basename, bids_root, sub, ses,
-                                    kind):
+def _get_bids_fpath_from_filesystem(*, bids_basename, bids_root, sub, ses,
+                                    kind, extension):
```

Comment:
is this private function needed anymore? If so, where? and why does it not use bids_path?

Reply:
never mind